drivers/Aqara/aqara-presence-sensor/config.yml

```
@@ -0,0 +1,5 @@
+name: 'aqara-presence-sensor'
```

Let's follow the naming convention [used in the other drivers](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/33f73765890b8e8ba80ec8be46f565ddec7e7b4a/drivers/Aqara/aqara-lock/config.yml#L1)

```suggestion
name: 'Aqara Presence Sensor'
```

---

I'll fix it.